action_server/frontend/src/lib/requestData.ts

```diff
@@ -333,8 +333,32 @@ export const collectOAuth2Status = async (
     isPending: true,
     data: undefined,
   });
-  const data = await loadAsync(`${baseUrl}/oauth2/status`, 'GET', {
+  const data = await loadAsync<any>(`${baseUrl}/oauth2/status`, 'GET', {
     params,
   });
   setLoaded(data);
 };
+
+export const cancelRun = async (runId?: string, requestId?: string) => {
+  if (!runId && !requestId) {
```

Comment:
same - if required, should arguments be made not optional as well? 

If some client deals with potetinal `undefined` values to provide to this function, client could go
```
cancelRun(myPossibleUndefined1 || '', myPossibleUndefined2 || '');
```

Reply:
It's the same thing, one of them is required while the other is optional.